README.md

```diff
@@ -2,19 +2,20 @@
 [![Build Status](https://travis-ci.org/optimizely/objective-c-sdk.svg?branch=master)](https://travis-ci.org/optimizely/objective-c-sdk/)
 [![Apache 2.0](https://img.shields.io/github/license/nebula-plugins/gradle-extra-configurations-plugin.svg)](http://www.apache.org/licenses/LICENSE-2.0)
 
-This repository houses the Objective-C SDK for Optimizely X Mobile.
+This repository houses the Optimizely mobile and OTT experimentation SDKs.
```

Comment:
super nit: might want to capitalize Mobile since it's the product Optimizey Mobile